src/Client/Updater.php

```diff
@@ -24,17 +23,11 @@
  *
  * @package Tuf\Client
  */
-class Updater
+class Updater implements LoaderInterface
```

Comment:
Updater is a loader itself because it is, fundamentally, responsible for confirming that the contents of a stream conform to a known set of trusted hashes before passing that stream on. It was already doing this, but implementing LoaderInterface makes it official.